### ISSUE TITLE
Feature: Let CodeBuddies vote on hangout topics they would like someone to host.

### ISSUE THREAD

**Issue by RuubixO** [codebuddies/codebuddies#1119]
Do you want to request a feature or report a bug?
Feature.

What is the current behavior?
Currently not available through slack or the app.

What is the expected behavior?
Have a place to propose desired topics people would like to be guided through like a 'most wanted' for code topics, and then make it visible to other members, who can add their votes to a specific topic, if they like. There should also be a way/button where any member can accept as a host/sponsor, which then automatically creates a hangout, or pushes that host/sponsor to the appropriate page to flesh out the rest of the details.

Which browser / OS are affected by this issue? (An *isolated* way to reproduce the bug)
Fedora 29 with latest Chrome and Firefox.

**Comment by lpatmo** [comment]
Missing from this screenshot: 
- When a user thumbs up the original post, they should see a popup asking for their timezone and general availability. Would that make sense?

Then, when a user clicks into the thread, they'll see everyone else who expressed interest (and their timezones/availability), and be able to reply and subscribe to the thread (feature that already exists).

**Comment by RuubixO** [comment]
Yes! This would be awesome. From the flip side: those who start these posts
should be able to invite other CodeBuddies with whom they want to see this
post. And, maybe to avoid spam, that they can only do so many invites every
day for any given user, just to keep the noise down.

**Comment by RuubixO** [comment]
I would also consider adding some filtering options for the search. You could have it dynamically add slack channels as tags that can be added to posts to help other people find them. It could also be good to let users add in custom tags if they're not already present. This could make finding specific CB-Connect proposals more effective.

**Comment by RuubixO** [comment]
I also think that, along with a selected channel, that new Discussion-posts should always end up in the Announcements channel, too. I frequently go there to look at wider community updates and we might have fellow members miss out on events they otherwise wouldn't see because they usually live in specific channels separate from where the Discussion-post is being advertised.

**Comment by ideaguy3d** [comment]
What's going on with this? Is anyone building it?

**Comment by lpatmo** [comment]
@ideaguy3d This'll probably get folded into V3 of the website (see: https://github.com/codebuddies/codebuddies/issues/1136)

In the meantime, we do have a recurring Slack reminder going out every Friday that's testing out this idea:

![image](https://user-images.githubusercontent.com/4512699/61196450-8becf680-a683-11e9-8709-03c91a313329.png)
